### PR TITLE
Add new method for add User accounts 

### PR DESCRIPTION
**Pull request by acidmax76**
1) add Accounts user
2) add param Account into Portfolio method

### CODE REVIEW

File: src/TIClient.php [hunk 1]
```diff
@@ -15,444 +15,555 @@
...
-        {
+        $currs  = [];
+        $params = [];
+        if ($account) {
+            $params = ['brokerAccountId' => $account->getBrokerAccountId()];
+        }
+        $response = $this->sendRequest(
+          "/portfolio/currencies",
+          "GET",
+          $params ? $params : []
```
Comment (samizdam) [inline src/TIClient.php]: `$params` уже инициализирован пустым массивом, эта проверка не имеет смысла.

**Comment by acidmax76** [comment]
add to code pls )

File: src/TICommission.php [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+    public function getCurrency()
+    {
+        return $this->currency;
+    }
+
+    public function getValue()
+    {
+        return $this->getValue();
+    }
```
Comment (jamesRUS52) [inline src/TICommission.php]: I think you mean thomethink like this
```php
return $this->value;
```

Comment (jamesRUS52) [inline src/TICommission.php]: I think you mean somethink like this
```php
return $this->value;
```

File: src/TIOperation.php [hunk 3]
```diff
@@ -31,12 +30,11 @@ class TIOperation {
...
     private $operationType;
     
-    function __construct($id, $status, $trades, $commisionCurrency, $commisionValue, $currency, $payment, $price, $quantity, $figi, $instrumentType, $isMarginCall, $date, $operationType) {
+    function __construct($id, $status, $trades, $commission , $currency, $payment, $price, $quantity, $figi, $instrumentType, $isMarginCall, $date, $operationType) {
         $this->id = $id;
         $this->status = $status;
         $this->trades = $trades;
-        $this->commisionCurrency = $commisionCurrency;
-        $this->commisionValue = $commisionValue;
+        $this->commission = $commission;
```
Comment (jamesRUS52) [inline src/TIOperation.php]: You have to remove getCommisionCurrency and getCommisionValue also
and create getComission method

File: src/TICommission.php (hunk 2 above)
Comment (acidmax76) [reply src/TICommission.php]: fuc..ing PHPStorm code generator )))

File: src/TIOperation.php (hunk 3 above)
Comment (acidmax76) [reply src/TIOperation.php]: yes

**Comment by acidmax76** [comment]
check pls

**Pull request merged**